src/app/core/services/data.service.ts

```diff
@@ -40,6 +40,19 @@ const isRfc3339StringDate = (date: Date | string): boolean => {
   }
 };
 
+// Translate units from sk metadata to appropriate type category
+const typeFromUnits = (units: string): string => {
+  if (!units) {
+    return undefined;
+  }
+  // So far the only thing that has units and isn't a number is a Date.
+  if (units === "RFC 3339 (UTC)") {
+    return "Date";
+  } else {
```

Comment:
Do we need this else? Not sure what's more readable...

```suggestion
  if (units === "RFC 3339 (UTC)") {
    return "Date";
  }
  return "number";
```

Reply:
Yeah, good call.